onsite/lucxor/cli.py

```
@@ -322,9 +336,178 @@ def __init__(self):
+        
+        # Fallback: use current score type and try to determine direction
+        higher_score_better = True
+        if hasattr(pep_ids[0], "isHigherScoreBetter"):
+            higher_score_better = pep_ids[0].isHigherScoreBetter()
+        elif hasattr(pep_ids[0], "getHigherScoreBetter"):
+            higher_score_better = pep_ids[0].getHigherScoreBetter()
+        
+        self.logger.warning(f"Using fallback score type: {current_score_type} (higher_better={higher_score_better})")
+        return current_score_type, higher_score_better
+    
+    def get_psm_score(self, pep_id: PeptideIdentification, hit, score_type: str, higher_score_better: bool) -> float:
```

Ruff (0.14.13)

Unused method argument: `pep_id`

(ARG002)